src/main/java/org/sunbird/workflow/service/impl/NotificationServiceImpl.java

```diff
@@ -78,7 +83,7 @@ public void sendNotification(WfRequest wfRequest) {
 		} catch (JsonProcessingException e) {
 			e.printStackTrace();
 		}
-		if (!ObjectUtils.isEmpty(wfStatus.getNotificationEnable()) && wfStatus.getNotificationEnable()) {
+		if (!ObjectUtils.isEmpty(wfStatus.getNotificationEnable()) && wfStatus.getNotificationEnable() && Constants.APPROVED.equalsIgnoreCase(wfStatus.getState())) {
```

Comment:
Do not check the status here, so that notifications are sent on request created, approved or rejected.

Reply:
If I removed this condition then user will get email two time when created the request and when approve the request , is it ok ?